### PR TITLE
Bug 1212776 - Enable bookmarks synchronization. r=yzen, michielbdjong

### PR DESCRIPTION
**Pull request by ferjm**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=dfb9d3391fe054eb68b0fe6e49b629f0c61bcb55

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=36ddc3eb61e5c1f8047aac241f82f0cde184292d

**Comment by michielbdejong** [comment]
Changing 7 files is not enough, you need to edit [one more](https://github.com/mozilla-b2g/gaia/blob/master/apps/settings/locales/settings.en-US.properties#L840-L841 ). ;)

**Comment by michielbdejong** [comment]
Actually, it would be good to add a link to this PR into a code comment, so next time we want to add a collection we have a checklist to refer to.

r+ for my part.

File: apps/settings/elements/firefox_sync.html [hunk 1]
```diff
@@ -50,6 +50,12 @@ <h2 data-l10n-id="fxsync-collections">Collections</h2>
       </header>
       <ul role="presentation">
         <li role="presentation">
+          <gaia-switch class="fxsync-collections-bookmarks"
+                       name="sync.collections.bookmarks.enabled">
+            <label data-l10n-id="fxsync-collections-bookmarks">Bookmarks</label>
```
Comment (yzen) [inline apps/settings/elements/firefox_sync.html]: We are not putting string into HTML any more (just the l10n-id data attr). Could you remove them from this template.

File: apps/settings/test/unit/_firefox_sync.html [hunk 2]
```diff
@@ -47,14 +45,20 @@ <h1 data-l10n-id="fxsync"></h1>
   <header>
     <h2 data-l10n-id="fxsync-collections">Collections</h2>
   </header>
-  <ul>
-    <li>
+  <ul role="presentation">
+    <li role="presentation">
+      <gaia-switch class="fxsync-collections-bookmarks"
+                   name="sync.collections.bookmarks.enabled">
+        <label data-l10n-id="fxsync-collections-bookmarks">Bookmarks</label>
```
Comment (yzen) [inline apps/settings/test/unit/_firefox_sync.html]: Here as well.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=485f0609763a59b1d542444e6997c2601485dced

**Pull request merged**